src/setup/rgw.py

```diff
@@ -132,6 +136,13 @@ def config(conf: MappedConf):
     def start(conf: MappedConf, index: str):
         """Create rgw admin user and start rgw service."""
 
+        # Before creating user/starting service,Verify backend store value=motr in rgw config file.
+        config_file = Rgw._get_rgw_config_path(conf)
+        Rgw._load_rgw_config(Rgw._rgw_conf_idx, f'ini://{config_file}')
+        if not Conf.get(Rgw._rgw_conf_idx, 'client>rgw backend store') == BACKEND_STORE:
+            raise SetupError(errno.EINVAL,
+                f'In {config_file}, "client>rgw backend store" key value should be "{BACKEND_STORE}".')
```

Comment:
Say 'Supported rgw backend store are ["{BACKEND_STORE}"], currently configured one is <whatever set vaue>

Above supported store print as list so we can add to that list in future if needed

Reply:
Done.